src/Components/Locations.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import { Component } from 'react';
+import "../App.css";
+
+class Locations extends Component {
+    state = {
+        locations: [],
+        showLocations: false
+    }
+
+    getLocations = async () => {
+        try {
+            const { data } = await axios.get(
+                'https://ghibliapi.herokuapp.com/locations'
```

Comment:
It is better to make this axios call one time in the component did mount than it is to make it every single time you click the button. From there you can just toggle between showing the list or not